samples/gltf/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Nov 20 10:27:45 PST 2017
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+#Wed May 06 20:41:57 EDT 2020
+distributionBase=GRADLE_USER_HOME

```

Comment:
I'm a little bit of a stickler for these line endings.  I've always used core.autocrlf=input and tried to stick to just LF endings.  Do you mind fixing these ^M line endings?

Reply:
Good catch and thanks for the git help